### PR TITLE
Add pgvectorizer

### PR DESCRIPTION
**Pull request by cevian**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 <!-- do not remove -->
 
+## 0.0.2
+
+Add ability to info start and end date from filters
```
Comment (cevian) [inline CHANGELOG.md]: This is an old fixup

Comment (jgpruitt) [inline CHANGELOG.md]: "infer"?

**Review by jgpruitt — CHANGES_REQUESTED** [review]
Consider using a static database function returning a constant for the advisory prefix to make it less likely for users to either use the same prefix for two different work queues or different prefixes for the same work queue for multiple vectorizors

File: nbs/01_pgvectorizer.ipynb [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+    "\n",
+    "        with psycopg2.connect(self.service_url) as conn:\n",
+    "            with conn.cursor() as cursor:\n",
+    "                cursor.execute(f\"\"\"\n",
+    "                    SELECT to_regclass('{self.schema_name}.{self.work_queue_table_name}')::oid; \n",
+    "                \"\"\")\n",
+    "                table_oid = cursor.fetchone()[0]\n",
+    "            \n",
+    "        with psycopg2.connect(self.service_url) as conn:\n",
+    "            with conn.cursor(cursor_factory=psycopg2.extras.DictCursor) as cursor:\n",
```
Comment (jgpruitt) [inline nbs/01_pgvectorizer.ipynb]: reuse the existing connection and potentially the existing cursor too

File: nbs/01_pgvectorizer.ipynb [hunk 3]
```diff
@@ -0,0 +1,388 @@
...
+    "            with conn.cursor() as cursor:\n",
+    "                cursor.execute(f\"\"\"\n",
+    "                    SELECT to_regclass('{self.schema_name}.{self.work_queue_table_name}') is not null; \n",
+    "                \"\"\")\n",
+    "                table_exists = cursor.fetchone()[0]\n",
+    "                if table_exists:\n",
+    "                    return\n",
+    "        \n",
+    "        with psycopg2.connect(self.service_url) as conn:\n",
+    "            with conn.cursor() as cursor:\n",
```
Comment (jgpruitt) [inline nbs/01_pgvectorizer.ipynb]: reuse the existing connection and potentially the existing cursor too

**Review by jgpruitt — CHANGES_REQUESTED** [review]

**Review by jgpruitt — APPROVED** [review]

**Pull request merged**